SSA/Projects/InstCombine/LLVM/Enumerator.lean

```diff
@@ -184,7 +184,7 @@ def generateRawSemantics : IO Unit := do
   rows := rows.append (binopRows "and"  (fun w a b => InstCombine.Op.denote (.and w)  [a,b]ₕ))
   rows := rows.append (binopRows "or"   (fun w a b => InstCombine.Op.denote (.or w)   [a,b]ₕ))
   rows := rows.append (binopRows "xor"  (fun w a b => InstCombine.Op.denote (.xor w)  [a,b]ₕ))
-  rows := rows.append (binopRows "add"  (fun w a b => InstCombine.Op.denote (.add w)  [a,b]ₕ))
+  rows := rows.append (binopRows "add"  (fun w a b => InstCombine.Op.denote (.add w false false)  [a,b]ₕ))
```

Comment:
Is this change not unnecessary, since the flags both have `false` as default value?